docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    coala also has some of its repositories on
```

Comment:
better preserve the style, that is already used. So frame this like `We also have some of our repositories.....`

Reply:
anyway coala always starting with a lower case is creating ambiguity on the sentence start